aspnet/mvc/models/formatting.rst

```diff
@@ -1,8 +1,217 @@
-.. include:: /../common/stub-topic.txt
+Formatting Response Data
+========================
 
-|stub-icon| Formatting
-======================
+By `Steve Smith`_
 
-.. include:: /../common/stub-notice.txt
+ASP.NET Core MVC has built-in support for formatting response data, using fixed formats or in response to client specifications.
 
-.. _issue: https://github.com/aspnet/Docs/issues/122
+.. contents:: Sections
+    :local:
+    :depth: 1
+
+`View or download sample from GitHub <https://github.com/aspnet/Docs/tree/master/mvc/models/formatting/sample>`_.
+
+Format-Specific Action Results
+------------------------------
+
+Some ActionResult options are specific to a particular format, such as JsonResult and ContentResult (note strings are simply returned as text). Actions can return specific results that are always formatted in a particular manner. For example, returning a ``JsonResult`` will return JSON-formatted data, regardless of client preferences. Likewise, returning a ``ContentResult`` will return plain text formatted string data.
```

Comment:
>(note strings are simply returned as text)

I'm missing the point of this. An action method with a string return type returns text?